tests/model_hub_tests/pytorch/common.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from optimum.intel import OVModelForCausalLM
+from pathlib import Path
+from openvino._pyopenvino.op import _PagedAttentionExtension
+from openvino._offline_transformations import paged_attention_transformation
+import openvino as ov
+
+def get_models_list(file_name: str):
+    models = []
+    with open(file_name) as f:
+        for model_name in f:
+            model_name = model_name.strip()
+            # skip comment in model scope file
+            if model_name.startswith('#'):
+                continue
+            models.append(model_name)
+    return models
```

Comment:
we already have such functionality: https://github.com/openvinotoolkit/openvino/blob/master/tests/model_hub_tests/models_hub_common/utils.py#L13
please re-use it.
We need to have common format for model files

Reply:
Ok, reused.